private/projects/nx-monorepo-project.ts

```diff
@@ -17,16 +17,21 @@ export class NXMonorepoProject extends PDKProject {
       name: "nx-monorepo",
       keywords: ["aws", "pdk", "jsii", "projen"],
       repositoryUrl: "https://github.com/aws/aws-prototyping-sdk",
-      devDeps: ["projen", "nx"],
+      devDeps: ["projen", "nx", "@types/fs-extra", "@types/semver"],
       peerDeps: ["projen"],
-      bundledDeps: ["@nrwl/devkit"],
+      bundledDeps: ["@nrwl/devkit", "fs-extra", "semver"],
       stability: Stability.STABLE,
     });
 
     this.compileTask.exec(
       'rsync -a ./src/** ./lib --include="*/" --include="**/*.js" --exclude="*" --prune-empty-dirs'
     );
 
-    this.package.addBin({ "nx-dir-hasher": "./scripts/nx-dir-hasher.js" });
+    this.package.addBin({ "pdk@nx-dir-hasher": "./scripts/nx-dir-hasher.js" });
```

Comment:
any reason why we are prefixing this with `pdk@`?

Reply:
Since these are bins the are available in PATH, and want to have clear scope for PDK based bin commands so not confused with other `nx-*` commands, and similarly with `pnpm-*` commands. The `@` is supported in filename cross-env so provides nice namespace separator.